### ISSUE TITLE
Dynamic drag image during drag

### ISSUE THREAD

**Issue by ddenev** [rlemaigre/Easy-DnD#30]
Hi,

Is is possible to have a dynamic drag image (one that can be changed during the drag operation)?

Here is an example of what I mean - https://marcojakob.github.io/dart-dnd/custom_avatar/

Best regards.

**Comment by rlemaigre** [comment]
That is something I had not envisioned. The drag image may only be defined using a slot on Drop components. But I suppose you could use the dragenter, dragleave and dragover events to do this yourself, or use the computed provided by the DragAwareMixin. But it isn't convenient.

**Comment by ddenev** [comment]
> But I suppose you could use the dragenter, dragleave and dragover events to do this yourself

Yes but the idea here is a little bit different - if you noticed in the example above, the image changes during the drag move (I guess based on distance from source) and does not depend on the drop zone.

So I guess this could be a nice enhancement indeed.

**Comment by rlemaigre** [comment]
No what I mean is to create the drag images yourself, animate them and swap between them, just like I do. But it's not convenient.

What I should do is allowing several drag-image slots like drag-image:a, drag-image:b, and so on...and allow to user to supply a function that returns which one should be currently in use (a or b or...).
